bundle-workflow/Pipfile

```diff
@@ -12,6 +12,10 @@ isort = "~=5.9"
 flake8 = "~=3.9"
 mypy = "~=0.9"
 pytest = "*"
+boto3 = "1.18.31"
+boto3-stubs = "1.18.34"
+mypy-boto3-s3 = "1.18.29"
```

Comment:
Instead of this, we can install it like `boto3-stubs[s3]` and then later import `from mypy_boto3_s3 import S3Client, S3ServiceResource`.  We won't need `boto3-stubs` then if we are using boto3 only for S3.
Ref: https://pypi.org/project/mypy-boto3-s3/